scripts/gdb/lvglgdb/value.py

```diff
@@ -0,0 +1,31 @@
+import gdb
+
+from typing import Optional
+
+
+class Value(gdb.Value):
+    def __init__(self, value: "gdb.Value | Value"):
```

Comment:
Is the type hint compatible with python 3.8?

For some version of gdb, it's required. Please check if `Union[gdb.Value, Value]` is better.

Reply:
<img width="1045" alt="image" src="https://github.com/user-attachments/assets/bb7738e3-6ead-438b-a38a-aadfc51dc7e7">

Reply:
> Is the type hint compatible with python 3.8?
> 
> For some version of gdb, it's required. Please check if `Union[gdb.Value, Value]` is better.

`Value` can't be used, because we are in `Value`